<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLSegmentEditorWidget</class>
 <widget class="qMRMLWidget" name="qMRMLSegmentEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qMRMLSegmentEditorWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_MasterVolume">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLScalarVolumeNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="noneDisplay">
        <string>Select master volume to enable editing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_MasterVolume">
       <property name="text">
        <string>Master volume:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_Segmentation">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLSegmentationNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Segmentation">
       <property name="text">
        <string>Segmentation:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="AddSegmentButton">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Add segment</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerSegmentationsModuleWidgets.qrc">
         <normaloff>:/Icons/Add.png</normaloff>:/Icons/Add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveSegmentButton">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Remove selected</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerSegmentationsModuleWidgets.qrc">
         <normaloff>:/Icons/Remove.png</normaloff>:/Icons/Remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MakeModelButton">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Make model</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerSegmentationsModuleWidgets.qrc">
         <normaloff>:/Icons/MakeModel.png</normaloff>:/Icons/MakeModel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qMRMLSegmentsTableView" name="SegmentsTableView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="EffectsGroupBox">
     <property name="title">
      <string>Effects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="OptionsGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="ActiveEffectLabel">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_ActiveEffectText">
        <property name="text">
         <string>Active effect:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="HelpLabel">
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tooltip is populated by the effects</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../qSlicerSegmentationsModuleWidgets.qrc">:/Icons/Help.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QFrame" name="EffectsOptionsFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>4</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentsTableView</class>
   <extends>QWidget</extends>
   <header>qMRMLSegmentsTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerSegmentationsModuleWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qMRMLSegmentEditorWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_Segmentation</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSegmentEditorWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_MasterVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MRMLNodeComboBox_Segmentation</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>SegmentsTableView</receiver>
   <slot>setSegmentationNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
